requirements/requirements.in

```
@@ -30,3 +30,4 @@ gordo-dataset~=2.3.0
 jeepney>=0.6
 packaging~=20.7
 pydantic~=1.7.3
+gordo.client~=0.1.5
```

Lets it be a future version:
```suggestion
gordo.client~=0.2.0
```

---

fixed